app/views/cases/_button_group.html.erb

```diff
@@ -1,13 +1,13 @@
 <div class = "btn-group btn-group-justified">
 	<% if user_signed_in? %>
 		<% if current_user.following?(@this_case) %>
-		    <%= link_to 'Unfollow', case_follow_path(@this_case, @follow_id), :confirm => "Are you sure you want to unfollow this case?", :method => :delete, class: "btn btn-primary.outline", id: "unfollow-button" %>
+		  <%= button_to 'Unfollow', follows_case_path(@this_case), method: :delete, data: { confirm: "Are you sure you want to unfollow this case?" }, form_class: "btn btn-primary btn-lg", id: "unfollow-button" %>
```

Comment:
@rlgreen91 `button_to` has been around since 4.2. Is this an essential piece of upgrading to 5.0, or is this a separate kind of upgrade?

Reply:
Part of upgrading.  Some of the changes I made ended up affecting how we handled the path for following cases.  That, in turn, led me to making the change to the path here, where I realized that eh, technically we shouldn't be using a link styled as a button to perform a post - it should actually be a button.  So I changed that here as part of it since I was already there.

Ironically enough, it was trying to fix the styling of the "unfollow" button that prompted my idea about the CSS fix.  The other two were fairly straightforward to change.